src/shared/case-editor/case-create.component.spec.ts

```diff
@@ -0,0 +1,339 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { DebugElement, NO_ERRORS_SCHEMA, Component, Input, EventEmitter, Output } from '@angular/core';
+import createSpyObj = jasmine.createSpyObj;
+import { CasesService } from '../cases/cases.service';
+import { CaseCreateComponent } from './case-create.component';
+import { CaseEventTrigger, Draft, CaseDetails, CaseEventData, DRAFT_PREFIX } from '../domain';
+import { createCaseEventTrigger } from '../fixture/shared.fixture';
+import { DraftService } from '../draft';
+import { AlertService } from '../alert';
+import { of, Observable } from 'rxjs';
+import { HttpError } from '../http';
+
+@Component({
+  selector: 'ccd-case-edit',
+  template: ``
+})
+class CaseEditComponent {
```

Comment:
Yes it probably could. The test was created as a copy of another existing test for a component already existing in case-editor dir. I will amend to mock and change the other tests too.